kola/harness.go

```diff
@@ -109,8 +109,8 @@ func filterTests(tests map[string]*register.Test, pattern, platform string, vers
 			continue
 		}
 
-		// Check the test's minVersion when running more then one test
-		if version.LessThan(t.MinVersion) && t.Name != pattern {
+		// Check the test's min and max versions when running more then one test
+		if t.Name != pattern && t.MinVersion.LessThan(t.MaxVersion) && (!t.MinVersion.LessThan(version) || !version.LessThan(t.MaxVersion)) {
```

Comment:
Ohhhhh, I see what you are saying. The reason I have it here is so that a Min and Max version of 0.0.0 (the default) still get executed. I guess I could trigger the panic in Register, and then check against 0.0.0 in this section.